spec/support/acceptance/post_tasks.rb

```diff
@@ -0,0 +1,18 @@
+require 'common_events_library'
+
+PE_CONSOLE = ENV['PE_CONSOLE']
+NUM_TASKS = ENV['NUM_TASKS'] || 5
+USERNAME = ENV['PT_PE_USERNAME'] || 'admin'
+PASSWORD = ENV['PT_PE_PASSWORD'] || 'pie'
+
+raise 'usage: PE_CONSOLE=<fqdn> post_tasks.rb' if PE_CONSOLE.nil?
+
+orchestrator = Orchestrator.new(PE_CONSOLE, USERNAME, PASSWORD, ssl_verify: false)
```

Comment:
This is the line that's failing the acceptance tests. These parameters are now named keyword arguments to account for the possibility of using either username/pass authentication or token authentication. Take a look at line 89 of the collector script for how to create this class now.